src/package-linker.js

```diff
@@ -137,6 +138,15 @@ export default class PackageLinker {
         phantomFiles.push(path.join(dest, file));
       }
 
+      try {
+        let obj = await fs.readJson(dest + "\\package.json");
```

Comment:
The path shouldn't contain `\\` as that is only valid for Windows. This should use `path.join(dest, 'package.json')` instead to work for all OSes.